Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -122,7 +121,7 @@ macro popQuad(reg, scratch)
     if ARM64 or ARM64E
         pop scratch, reg
     elsif X86_64
-        pop reg
+        pop reg, scratch
```

Comment:
I think we could make popQuad(reg) be ldr reg, sp, 16 (I would probably call it load64Inc in offlineasm) we wouldn't need to have a scratch. This could probably be a general opcode for all the CPUs or arm specific.